jbpm-test-util/src/main/java/org/jbpm/test/util/Version.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.test.util;
+
+public class Version implements Comparable<Version> {
+
+    Integer major;
+    Integer minor;
+
+    public Version(String version) {
+        String[] parts = version.split("\\.");
+        this.major = Integer.parseInt(parts[0]);
+        this.minor = Integer.parseInt(parts[1]);
+    }
+
+
+    public int getMajor() {
+        return major;
+    }
+
+    public int getMinor() {
+        return minor;
+    }
+
+    @Override
+    public int compareTo(Version other) {
+        if (major.equals(other.major)) {
+            return minor.compareTo(other.minor);
+        }
+        return major.compareTo(other.major);
+    }
+
+    @Override
+    public boolean equals(Object obj) {
```

Comment:
hasCode method should be implemented, but up to you as this is a test class.

Reply:
done